src/azul/logging.py

```diff
@@ -6,9 +6,39 @@
 )
 
 
+# lambda python bootstrap.py line 284 - to add format attribute of aws_request_id
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, app: AzulChaliceApp = None):
+        try:
+            # From https://github.com/aws/aws-lambda-base-images/tree/python3.8
+            # Imports global variable present in lambda environment
+            from bootstrap import (  # noqa:AZUL141
+                _GLOBAL_AWS_REQUEST_ID,
+            )
+        except ImportError:
+            # Arbitrary UUID for local testing in place of non-existent lambda env var
```

Comment:
Comments should answer `why` something is happening. I can read that this block is executed if there is an ImportError.

I'll let you come up with something.